test/cli/commands_test.py

```diff
@@ -407,37 +437,60 @@ def test__cli__command_fix_stdin_safety():
 
 
 @pytest.mark.parametrize(
-    "sql,exit_code",
+    "sql,exit_code,params,output_contains",
     [
-        ("create TABLE {{ params.dsfsdfds }}.t (a int)", 1),  # template error
-        ("create TABLE a.t (a int)", 0),  # fixable error
-        ("create table a.t (a int)", 0),  # perfection
-        ("select col from a join b using (c)", 1),  # unfixable error (using)
+        (
+            "create TABLE {{ params.dsfsdfds }}.t (a int)",
+            1,
+            "-v",
+            "Fix aborted due to unparseable template variables.",
+        ),  # template error
+        ("create TABLE a.t (a int)", 0, "", ""),  # fixable error
+        ("create table a.t (a int)", 0, "", ""),  # perfection
+        (
+            "select col from a join b using (c)",
+            1,
+            "-v",
+            "Unfixable violations detected.",
+        ),  # unfixable error (using)
```

Comment:
@nolanbconaway I enhanced these tests to get the coverage back up to 100%

Reply:
Yeah thx! my setup of adding new adhoc strings to test wasnt exactly scalable and this is